### ISSUE TITLE
[docs] Create documentation for Code of conduct

### ISSUE THREAD

**Issue by andrewnicols** [moodle/devdocs#47]
### What is the Moodle feature that needs documenting?

We should make sure that:

- we have a code of conduct
- we have all read it
- require that it be signed for contributions?

As far as I'm aware we don't really have a code of conduct at the moment. We do have a Forums code of conduct for taking part in the Moodle forums, but it is out-dated and does not apply clearly to this project.

One CoC that we may want to consider adopting is the Contributor Covenant (https://www.contributor-covenant.org/), which is widely used within the Open Source community.

### Is this documentation specific to a Moodle version?

No

### Are you able to provide a patch for this?

No (Default)

**Comment by andrewnicols** [comment]
Addign @bjs1000  and @abgreeve as some of the assignees of this issue as this will require further discussion and affects the whole Moodle LMS project.

**Comment by sarjona** [comment]
I've prepared a proposal for the Code of Conduct, based on the [Contributor Coverant](https://www.contributor-covenant.org/). Basically, the only change I've applied is to remove the "Enforcement Guidelines" section.

Apart from getting some feedback from @bjs1000, @abgreeve and any other people from the community, we also need to decide on the contact method defined in the Enforcement section.

I've also found https://docs.moodle.org/dev/Working_with_the_Community and I was wondering whether we should also include the information there or not... Thoughts?
